File: qe_selenuim_python/Activity4.py
from selenium import webdriver
from selenium.webdriver.common.by import By

driver = webdriver.Chrome()
driver.get("https://training-support.net/webelements/target-practice")
driver.maximize_window()

print(driver.title)

third_header = driver.find_element(By.XPATH, "(//h1 | //h2 | //h3 | //h4 | //h5)[3]")
print(third_header.text)

fifth_header = driver.find_element(By.XPATH, "(//h1 | //h2 | //h3 | //h4 | //h5)[5]")
print(fifth_header.value_of_css_property("color"))

purple_button = driver.find_element(By.CSS_SELECTOR, ".bg-purple-500")
print(purple_button.get_attribute("class"))

slate_button = driver.find_element(By.CSS_SELECTOR, ".bg-slate-500")
print(slate_button.text)

driver.quit()

File: qe_selenium_java/Activity4.java
package demos;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity4 {

    public static void main(String[] args) {

        WebDriver driver = new ChromeDriver();

        driver.get("https://training-support.net/webelements/target-practice");
        driver.manage().window().maximize();

        System.out.println(driver.getTitle());

        WebElement thirdHeader = driver.findElement(By.xpath("(//h3)[3]"));
        System.out.println(thirdHeader.getText());

        WebElement fifthHeader = driver.findElement(By.xpath("(//h3)[5]"));
        System.out.println(fifthHeader.getCssValue("color"));

        WebElement purpleButton = driver.findElement(By.id("purple"));
        System.out.println(purpleButton.getAttribute("class"));

        WebElement slateButton = driver.findElement(By.cssSelector("button.bg-slate-500"));
        System.out.println(slateButton.getText());

        driver.quit();
    }
}
